nipype/pipeline/plugins/base.py

```diff
@@ -222,6 +218,9 @@ def _clean_queue(self, jobid, graph, result=None):
 
         if self._status_callback:
             self._status_callback(self.procs[jobid], 'exception')
+		if result is None:
+			result={'result': None,
+					'traceback': '\n'.join(format_exception(*sys.exc_info()))}
```

Comment:
Looks like you're using tabs, not spaces. Do you think you could update this to use 4-space indentation? Also, there should be spaces around the `=`, and `'traceback'` should start on the same column as `'result'`.

(Sorry if this is a pain in your editor.)